packages/plugin-core/src/commands/MoveHeader.ts

```diff
@@ -130,21 +147,34 @@ export class MoveHeaderCommand extends BasicCommand<
    * @returns
    */
   private promptForDestination(opts: CommandInput) {
-    const lookupController = LookupControllerV3.create({
+    const lcOpts: LookupControllerV3CreateOpts = {
       nodeType: "note",
-      disableVaultSelection: true,
-    });
+      disableVaultSelection: opts?.useSameVault,
+      vaultSelectCanToggle: false,
+    };
+    const lookupController = LookupControllerV3.create(lcOpts);
     const lookupProvider = new NoteLookupProvider(this.key, {
-      allowNewNote: false,
+      allowNewNote: true,
       noHidePickerOnAccept: false,
     });
+    lookupProvider.registerOnAcceptHook(ProviderAcceptHooks.NewLocationHook);
+
+    let initialValue: string | undefined;
+    let nonInteractive: boolean = false;
+    if (opts?.dest) {
+      initialValue = opts.dest.fname;
+      nonInteractive = true;
+    } else if (opts?.initialValue) {
+      initialValue = opts.initialValue;
+      nonInteractive = true;
```

Comment:
`initialValue` shouldn't apply `nonInteractive` that should be taken from the passed in value

Reply:
good point. this was intended for easier testing and not really to expose the args to the users. but we should keep it consistent with lookup.